apps/acrs/behaviours/get-decision-email.js

```diff
@@ -0,0 +1,13 @@
+
+module.exports = superclass => class extends superclass {
+  saveValues(req, res, next) {
+    const save = super.saveValues(req, res, next);
+    const email = req.sessionModel.get('user-email');
+    if (req.form.values['is-decision-by-email'] === 'yes') {
+      req.sessionModel.set('decision-email', email);
+    } else {
+      req.sessionModel.set('decision-email', req.form.values['is-decision-by-email-detail']);
+    }
+    return save;
+  }
+};
```

Comment:
This behaviour looks identical to the `save-decision-email.js` behaviour. Are they both needed?

Reply:
Removed